concepticondata/references/references.bib

```diff
@@ -30,6 +30,17 @@ @article{Araujo1996
 	year = {1996},
 }
 
+@Article{Wang2021,
+  author    = {Xiaosha Wang and Yanchao Bi},
+  title     = {Idiosyncratic Tower of Babel: Individual Differences in Word-Meaning Representation Increase as Word Abstractness Increases},
+  doi       = {10.1177/09567976211003877},
+  pages     = {095679762110038},
```

Comment:
I didn't catch that earlier. The page numbers should be "1617-1635".

Reply:
@AnnikaTjuka Well spotted, thanks!